TypeScript/src/ux-elements/footer-nav/nav.tsx

```diff
@@ -41,7 +41,10 @@ export default function Nav() {
       </button>
       <button
         onClick={() => {
-          initializePage({ pageRoute: 4 }, page?.setPageNumber);
+          initializePage(
+            { pageRoute: PageIdentityEnum.FEED_PAGE },
+            page?.setPageIdentity
```

Comment:
do we need the ? here we took out from 16.

Reply:
Ah yeah we probably dont need any of them, either one of us can take them out in the next pr